api/proto/config/kayenta.proto

```diff
@@ -0,0 +1,41 @@
+syntax = "proto3";
+package proto.config;
+
+option go_package = "github.com/spinnaker/kleat/api/client/config";
+
+import "canary/aws.proto";
+import "canary/datadog.proto";
+import "canary/google.proto";
+import "canary/newrelic.proto";
+import "canary/prometheus.proto";
+import "canary/signalfx.proto";
+import "canary/stackdriver.proto";
+import "canary/gcs.proto";
+import "canary/s3.proto";
+
+// Configuration for the Kayenta microservice.
+message Kayenta {
+  ServiceIntegrations kayenta = 1;
+
+  message ServiceIntegrations {
+    Google google = 1;
+    canary.Stackdriver stackdriver = 2;
+    canary.Gcs gcs = 3;
+    canary.Prometheus prometheus = 4;
+    canary.Datadog datadog = 5;
+    Aws aws = 6;
+    canary.S3 s3 = 7;
+    canary.SignalFx signalfx = 8;
+    canary.NewRelic newrelic = 9;
+
+    message Google {
```

Comment:
So, this is a little annoying and I'm open to other ways of structuring the messages here. A halconfig canary.Google block is different than a Kayenta canary.Google block because we only need `enabled` and `accounts` to populate the Kayenta block (and derive the Stackdriver and GCS blocks from the other top-level fields). Similar story with AWS and S3. This will probably be clearer once I write the translation logic for hal --> kayenta, so let me know if this would be easier to review if I added that as a third commit to this PR.

Reply:
I think this makes sense, and is similar to what I did for authentication, where there is a `google` block under `Gate` that would never really mean anything else outside of Gate so I made it a nested message there.